lib/resources/packages.rb

```diff
@@ -23,12 +23,17 @@ class Packages < Inspec.resource(1)
     "
 
     def initialize(pattern)
-      @command = packages_command
-
-      return skip_resource "The packages resource is not yet supported on OS #{inspec.os.name}" unless @command
+      os = inspec.os
+      if os.debian?
+        @pkgs = Debs.new(inspec)
+      elsif %w{redhat suse amazon fedora}.include?(os[:family])
```

Comment:
I'd love it if we could align a bit more with Chef/Ohai in terms of platform / os / platform_family so it's not confusing for our users. Not something to address in this PR, but perhaps something we can consider for a future enhancement. Curious as to your thoughts.